Configure

```diff
@@ -1825,10 +1826,12 @@ EOF
                 $ddest = $unified_info{rename}->{$ddest};
             }
             foreach (@{$ordinals{$dest}}) {
-                my %known_ordinals =
+                my %known_ordinals = # TODO this should be configurable in build.info using ORDINALS
```

Comment:
Actually, *everything* about ORDINALS can be removed, both here and anywhere in `Configurations`.

Reply:
Do you mean that also util/lib*.num are not needed anymore, not even for building Windows DLLs? 
The better if so, because these often cause(d) extra work.
Hope that someone who is very familiar with this topic takes care of removing the unncessary pieces.

Reply:
Ah, no, not quite so unnecessary, sorry.  What's not needed any more is the `build.info` keyword `ORDINALS`.  It serves no purpose any more, as we've made the generation of corresponding .map / .def / .opt file explicit.

I will, however, clean out the stuff around `ORDINAL`, thanks.